packages/libs/react-components/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["heft-jest"],
+    "target": "es5",
```

Comment:
Can we use a more modern version, or are we leaving consumers behind. I'd argue that when you're using the `esm` variant, you will have a modern browser as well. Based on this table we could find one that's widely used for ESM variant: https://kangax.github.io/compat-table/es2016plus/ (ignoring IE11 of course)
Maybe a minimum of ES2017 would be doable.